leaks.go

```diff
@@ -69,12 +69,12 @@ func FindLeaks(options ...Option) error {
 	return fmt.Errorf("found unexpected goroutines:\n%s", stacks)
 }
 
-// VerifyNoLeaks calls FindLeaks and calls Error on the passed in TestingT if
+// VerifyNone calls Find and calls Error on the passed in TestingT if
```

Comment:
nit: "calls Find and calls Error" reads a bit like repeating the implementation. How about something like,

```
// VerifyNone marks the given TestingT as failed if any extra goroutines are
// found by Find.
```